### PR TITLE
feat : liked and watch later

### PR DESCRIPTION
**Pull request by anuj-kumary**
This PR includes:-

- Liked Video Functionality
- Add To Watch Later Functionality

Preview:- https://deploy-preview-4--techhub-react.netlify.app/

### CODE REVIEW

**Review by mohammadkhizerkhan — APPROVED** [review]
Good naming convention ,
Good to merge

File: src/components/Sidebar/Sidebar.jsx [hunk 1]
```diff
@@ -1,26 +1,27 @@
...
 export const Sidebar = () => {
   return (
     <>
       <aside className='sidebar'>
         <ul className='sidebar__menu'>
-          <li className='sidebar__link text active'>
-            <i className='fas fa-home'></i>Home
-          </li>
-          <li className='sidebar__link text'>
+          <Link to='/video' className='sidebar__link text active'>
```
Comment (vinc3nati) [inline src/components/Sidebar/Sidebar.jsx:10]: You can use `NavLink` for using activeClass method to toggle the css when the link is active

Comment (vinc3nati) [inline src/components/Sidebar/Sidebar.jsx]: You can use NavLink for using activeClass method to toggle the css when the link is active

**Review by vinc3nati — APPROVED** [review]
Rest all looks good

**Review by hrishi82 — APPROVED** [review]

File: src/components/Sidebar/Sidebar.jsx (hunk 1 above)
Comment (anuj-kumary) [reply src/components/Sidebar/Sidebar.jsx]: Thanks for the suggestion. I'll make this changes

**Pull request merged**